src/Moq/Protected/ProtectedAsMock.cs

```diff
@@ -255,7 +272,7 @@ protected override Expression VisitMember(MemberExpression node)
 				}
 				else
 				{
-					return node;
+					return base.VisitMember(node);
```

Comment:
&dagger;) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.